astronomer/providers/amazon/aws/sensors/redshift_cluster.py

```diff
@@ -27,45 +19,20 @@ def __init__(
     ):
         # TODO: Remove once deprecated
         if poll_interval:
-            self.poke_interval = poll_interval
+            kwargs["poke_interval"] = poll_interval
```

Comment:
shall we remove `poll_interval` from init and check kwargs["poll_interval"] in L21 this way code editor will probably not suggest this param

```python
if kwargs.get("poll_interval"):
    kwargs["poke_interval"] = kwargs["poll_interval"]
```

Reply:
good idea! just changed